index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Tribute Page</title>
+
```

Comment:
I would like to advise you to consider adding a code to display a "favicon" in your browser when the web page will be deployed for SEO matters.

Reply:
@danmainah thanks!... I did think about it, but I was too lazy haha.